scripts/measures/archived/scripts/get-stratification.js

```diff
@@ -24,19 +58,24 @@ function extractStrata(measure) {
     const components = component.populationCriteriaSection[0].component;
     const numeratorUuid = components.find(item => item.numeratorCriteria).numeratorCriteria[0].id[0].$.root;
     const stratList = [];
+    // Loops through a Stratifier Criteria's components. If it's not a supplemental data component,
+    // add the stratum.
     components.filter(item => item.stratifierCriteria).forEach((component, index) => {
       const supplementalDataComponent = component.stratifierCriteria[0].component;
-      if (supplementalDataComponent === undefined ||
-        supplementalDataComponent[0].measureAttribute[0].code[0].$.code !== 'SDE') {
+      if (_.isUndefined(supplementalDataComponent) ||
+        supplementalDataComponent[0].measureAttribute[0].code[0].$.code !== supplementDataType) {
         stratList.push(component.stratifierCriteria[0].id[0].$.root);
       }
     });
     if (stratList.length !== 0) {
-      retVal[numeratorUuid] = stratList;
+      const numeratorMap = {};
+      numeratorMap['numeratorUuid'] = numeratorUuid;
+      numeratorMap['strata'] = stratList;
+      returnValue.push(numeratorMap);
     }
   });
 
-  return retVal;
+  return returnValue;
```

Comment:
Oh, I should've realized retVal stood for returnValue! What I should've said is: can you rename this variable to something descriptive of what's being returned, e.g. `strataMaps`?

Reply:
👍 makes sense and its more clear.